build.groovy

```diff
@@ -39,7 +39,7 @@ else {
 		println("** Invoking build scripts according to build order: ${props.buildOrder}")
 		String[] buildOrderList = props.buildOrder.split(',')
 		String[] testOrderList;
-		if (props.runzTests == "True") { 
+		if (props.runzTests) { 
```

Comment:
It is enough ? Or should we have this ?
if (props.runzTests && props.runzTests .toBoolean())

Reply:
Agreed, we need to test the boolean value of it!